tests/oauth2/test_service_account.py

```diff
@@ -71,6 +71,7 @@ def test_constructor_no_universe_domain(self):
             SIGNER, self.SERVICE_ACCOUNT_EMAIL, self.TOKEN_URI, universe_domain=None
         )
         assert credentials._universe_domain == service_account._DEFAULT_UNIVERSE_DOMAIN
```

Comment:
do we need to check the private variable as well now?

Reply:
right, it's not necessary, deleted that